sample/simpleidp/index.js

```diff
@@ -39,7 +39,8 @@ app.use('/api/login', (req, res) => {
   let username = req.body.username
 
   // Here we do a simple hashed password check
-  crypto.pbkdf2(password, 'salt', 100000, 64, 'sha512', (err, derivedKey) => {
+  const salt = crypto.randomBytes(16)
```

Comment:
If you want per-password salts, instead of a global one, you should store this together with the hashed password? And use it for the verification-encoding?